src/fragments/lib/logging/ios/flushlogs/flushlogs.mdx

```diff
@@ -0,0 +1,13 @@
+If there are use cases where logs need to be sent immediately, you can use the `AWSCloudWatchLoggingPlugin` instance to force an automatic log flush.
+
+Add import statement to the `AWSCloudWatchLoggingPlugin`
```

Comment:
can we have the code snippet together for both the import and the plugin. No need to have them seperate I think right?

Reply:
We've had confusion in the past where customers didn't realize that the import statements are separate from the code sample, so we've started to separate them to reduce the confusion.  I'd like to keep it separate unless you feel strongly that it should be combined.

Reply:
That's ok - we can keep it as is